src/initgroups/__init__.py

```diff
@@ -12,4 +12,16 @@
 #
 ##############################################################################
 
-from _initgroups import initgroups
+import os
+import warnings
+
+
+_MESSAGE =(
+    "The standalone 'initgroups.initgroups' functinon is deprecated.  "
+    "Use 'os.initgroups' instead.")
+
+# We do not use DeprecationWarning because it is ignored by default.
+warnings.warn(_MESSAGE)
```

Comment:
I don't like this.

I understand why DeprecationWarnings were hidden by default (end-users can't do anything about them), and I don't think removing the use of a thin backwards-compatibility wrapper warrants overriding warning policy that way.

Reply:
OK, I'll acquiesce.